ddtrace/contrib/botocore/patch.py

```diff
@@ -8,7 +8,9 @@
 from typing import Any
 from typing import Dict
 from typing import List
+from typing import Literal
```

Comment:
TIL that Literal was added in Python 3.8

Sorry about that.

Instead of `Literal` and `Optional[Union[Literal["sqs"], Literal["sns"]]]` we should just do: `Optional[str]`.